entries/entry02.md

```diff
@@ -1,7 +1,24 @@
 # Entry 2
```

Comment:
Sources: 1
EDP: 1
Knowledge: 1
Skills: 1
Professionalism: 1

Great job! Just make sure you speak more about the skills.